.github/workflows/build-images.yml

```diff
@@ -84,10 +77,9 @@ jobs:
       uses: spilchen/switch-case-action@v2
```

Comment:
I can't make a comment on the exact line, but we set OWNER_LC a few lines that that we don't need anymore.